stars/default/name.fab

```diff
@@ -369,10 +401,15 @@
  12803|ο_Ari
  13165|π_Ari
  13702|ρ_Ari
+ 13579|ρ1_Ari
+ 13654|ρ2_Ari
+ 13702|ρ3_Ari
  13327|σ_Ari
  15627|τ1_Ari
  15737|τ2_Ari
+ 13209|c_Ari
   8544|1_Ari
+  8271|3_Ari*
```

Comment:
Why asterisk added here?

Reply:
According to [HR_515](https://en.wikipedia.org/wiki/HR_515) and [Table of stars with Flamsteed designations](https://en.wikipedia.org/wiki/Table_of_stars_with_Flamsteed_designations)

Reply:
Please remove asterisk from designations, because this file contains modern designations.